backend/src/main/kotlin/org/loculus/backend/service/submission/SubmissionDatabaseService.kt

```diff
@@ -345,11 +332,9 @@ open class SubmissionDatabaseService(
         if (numberInserted != 1) {
             throwInsertFailedException(submittedProcessedData, pipelineVersion)
         }
-
-        return newStatus
     }
 
-    private fun postprocessAndValidateProcessedData(
+    private fun postProcessAndValidateProcessedData(
```

Comment:
I'm not sure whether this change is for better or worse. I think a docstring here would really help as it's not immediately obvious from the name and arguments what this function does. It seems to do a status change, i.e. not be purse.

There might also be a misunderstanding. I _think_ this function name means postprocessing as a verb, not posting something, or doing something post=after "ProcessAndValidate"

Reply:
```suggestion
    private fun postprocessAndValidateProcessedData(
```

I think it's pretty clearly postprocessing that's meant, as the processed sequences are turned upper case among other things